README.md

```diff
@@ -43,6 +43,14 @@ modules:
       - "Could not connect to database"
       fail_if_not_matches_regexp:
       - "Download the latest version here"
+  http_post_2xx:
+    prober: http
+    timeout: 5s
+    http:
+      method: POST
+      headers:
+        Content-Type: application/json
+      body: '{}' # Content. Type must match the Content-Type header above
```

Comment:
will it help to merge if i'll add same PR without mentioned comment and without conflict, as author doesn't seem to answer any type of message?
thanks